tests/ZendTest/Form/Annotation/AnnotationBuilderTest.php

```diff
@@ -20,7 +20,7 @@ public function setUp()
         if (!defined('TESTS_ZEND_FORM_ANNOTATION_SUPPORT')
             || !constant('TESTS_ZEND_FORM_ANNOTATION_SUPPORT')
         ) {
-            $this->markTestSkipped('Enable TESTS_ZEND_FORM_ANNOTATION_SUPPORT to test annotation parsing');
+           // $this->markTestSkipped('Enable TESTS_ZEND_FORM_ANNOTATION_SUPPORT to test annotation parsing');
```

Comment:
CS checks fail on this file. Probably because of this line, but not really sure.